client/jetpack-connect/style.scss

```diff
@@ -656,3 +656,25 @@
 .jetpack-connect__navigation {
 	text-align: center;
 }
+
+.jetpack-connect__plan-info {
+	margin-top: 35px;
+	text-align: center;
+	width: 970px;
+}
+
+.jetpack-connect__plan-info-illustration {
+	margin: 30px 0 10px;
+	max-width: 25%;
+}
+
+.jetpack-connect__plan-info-buttons {
+	display: block;
+	margin-bottom: 20px;
```

Comment:
Doesn't feel right to have this space defined here; do we want to add it as a `padding-bottom` of the parent, so it will be there if we move the elements or append something after them?

Reply:
See below 👍